lib/hz2600/CallflowBuilder/Builder.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace CallflowBuilder;
+
+use CallflowBuilder\Node\AbstractNode; 
+
+class Builder {
```

Comment:
I wonder if you could change Callflow to have a `setFlow($root)` that called $root->build() inside the function:

```php
public function setFlow(AbstractNode $root) {
  $this->flow = $root->build();
}
```